src/main/java/org/embulk/input/FtpFileInputPlugin.java

```diff
@@ -68,6 +71,7 @@
         @Config("port")
         @ConfigDefault("null")
         public Optional<Integer> getPort();
+        public void setPort(Optional<Integer> port);
```

Comment:
I created 3 types default ports depends on FTP mode.
* FTP 21
* FTPS 990
* FTPES 21

I created this method to set port number from Java.
I think this method is unnecessary if there is only 1 default port type as you're saying.

```java
int defaultPort = FTP_DEFULAT_PORT;
if (task.getSsl()) {
    ...
    if (task.getSslExplicit()) {
        ...
        defaultPort = FTPES_DEFAULT_PORT;
        ...
    }
    else {
        ...
        defaultPort = FTPS_DEFAULT_PORT;
        ...
    }
}

if (!task.getPort().isPresent()) {
    task.setPort(Optional.of(defaultPort));
}
```